docs/src/Module.doc.js

```diff
@@ -36,6 +36,16 @@ card(
         type: 'string',
         required: true,
       },
+      {
+        name: 'extExpandedId',
+        type: '?string',
+        required: false,
```

Comment:
could you add a link to the example


use `href` & `id` to `card`

Reply:
Also, can we add a little description to the prop?

Reply:
added